src/components/Dashboard/DashboardStateProvider.js

```diff
@@ -54,7 +54,8 @@ function WithSubscription({ Provider, connectedAccount, children }) {
   } = useJurorDraftsNotRewardedSubscription(account)
 
   const fetching = balancesFetching || appealsFetching || jurorDraftsFetching
-  const errors = [...balanceErrors, ...appealErrors, jurorDraftsError]
+  const errors = [...balanceErrors, ...appealErrors]
+  jurorDraftsError && errors.push(jurorDraftsError)
```

Comment:
This is kind of weird to understand at a glance; might as well just make it an if or move it above and do an `jurorDraftsError ? [jurorDraftsError] : []`